vispy/visuals/markers.py

```diff
@@ -702,19 +692,18 @@ def scaling(self):
     @scaling.setter
     def scaling(self, value):
         scaling_modes = {
-            False: 0,
-            True: 1,
-            "fixed": 0,
-            "scene": 1,
-            "visual": 2,
+            False: "fixed",
+            True: "scene",
+            "fixed": "fixed",
+            "scene": "scene",
+            "visual": "visual",
         }
         if value not in scaling_modes:
             possible_options = ", ".join(repr(opt) for opt in scaling_modes)
             raise ValueError(f"Unknown scaling option {value!r}, expected one of: {possible_options}")
-        scaling_int = scaling_modes[value]
-        self.shared_program['u_scaling'] = scaling_int
         self._scaling = value
-        self._scaling_int = scaling_int
+        self._scaling_mode = scaling_modes[value]
```

Comment:
Is there any need for a separate `_scaling_mode` and `_scaling` now that an integer isn't based to the shader?

Reply:
If we want to retain the ability to do:

```py
marker.scaling = False
assert marker.scaling == False
```

Otherwise we'd get:
```py
marker.scaling = False
assert marker.scaling == 'fixed'
```

I'd be ok with the latter, but I thought you wanted to retain the previous behaviour in #2470?

Reply:
I think I was hoping for backwards compatibility with someone setting it to `False/True`. My comment here is to say that `self._scaling` and `self._scaling_mode` are now the same for users specifying the new flags. If someone is doing `assert marker.scaling == X` then I personally am OK with that failing now. But maybe I'm misremembering what I said before.

Reply:
yeah for the new flags those are identical, but I agree it's nicer to just ditch the true/false otherwise.

Reply:
Updated to only keep the new value.